sdk/storage/test-resources-post.ps1

```diff
@@ -1,6 +1,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+echo $Env:RESOURCE_GROUP
+echo $Env:ACCOUNT_NAME
```

Comment:
The `New-TestResources.ps1` script sets environment variables in DevOps but those environment variables only take effect in subsequent steps. DevOps itself cannot mutate environment variables for a running process which is why they're not available here. 

Pre and Post scripts are called with parameters that will give you what you need. Set up params to receive `$DeploymentOutputs` and `$ResourceGroup` and you'll be able to receive those. 

```pwsh
param(
  [string] $ResourceGroupName,
  [hashtable] $DeploymentOutputs,
  [string] $TenantId,
  [string] $TestApplicationId,
  [string] $TestApplicationSecret
)

Write-Host "Resource group: $ResourceGroupName" 
Write-Host "Account name: $($DeploymentOutputs['ACCOUNT_NAME']" 
```

## Use az CLI or Az PowerShell modules

If you're going to use the `az` CLI you'll need to authenticate it first. We do pass in `$TestApplicationId` etc. to help you do that but, if possible, use the `Az` PowerShell modules because those are already set up and authenticated. I think this one will do what you need. https://docs.microsoft.com/en-us/azure/storage/blobs/encryption-scope-manage?tabs=powershell#create-an-encryption-scope-protected-by-microsoft-managed-keys

Reply:
This works. Thanks @danieljurek !